lib/image/imagemagick.js

```diff
@@ -367,7 +382,63 @@ module.exports = function() {
 
         args.push('null:');
 
-        return spawnThen('convert', args, callback);
+        return spawnThen('convert', args, function (err) {
+          if (err) {
+            return callback(err);
+          }
+
+          if (!options.imagemin) {
+            return callback();
+          }
+
+          imagemin(resizedPaths, context.tempFolder + '/', {
+            plugins: [
+              imageminJpegRecompress(),
+              imageminMozjpeg(),
+              imageminPngquant({quality: '65-80'}),
+              imageminOptipng()
+            ]
+          })
+            .then(files => {
+            //= > [{data: <Buffer 89 50 4e …>, path: 'build/images/foo.jpg'}, …]
+              return callback(null);
+            })
+            .catch(err => {
+              console.log('ERROR', err);
+              // returning ok callback as this additionnal compression doesn't prevent images usage
+              return callback(err);
+            });
+        });
+      }
+
+      function convertSvg (context, callback) {
```

Comment:
This doesn't crop, so it's not really a fully valid uploadfs implementation for svg files. I'll talk to Ludo about whether he is using this and considers it important to release before we add time for that.

Reply:
Agreed. Let me know how to proceed. I guess the options are to remove this, or to implement, or leave the hack so it doesn't break existing implementations...

Reply:
Please remove svg for now so we can get this out the door to npm, however I have opened a new ticket to bring back the svg support plus cropping to match other formats.